csfieldguide/templates/interactives/iframe-interactive-base.html

```diff
@@ -0,0 +1,22 @@
+{% load static %}
```

Comment:
Why isn't this template inheriting from the base template?

Reply:
not sure that I understand why it would need to?